UPGRADE.md

```diff
@@ -51,6 +51,35 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 
 ## [From 7.0.0-beta2 to Unreleased]
 ### [shopsys/framework]
+- [#627 model service layer removal](https://github.com/shopsys/shopsys/pull/627)
+    - we moved business logic into entities, following instructions tells you what method was moved where, so you can upgrade your code and also tests.
```

Comment:
You wrote that you moved the logic to entities but you moved some methods to the facades and factories as well so it could be confusing.